ui/lib/sync/addon/components/secrets/page/overview.hbs

```diff
@@ -191,7 +191,11 @@
         >documentation</Hds::Link::Inline>
         to learn more.
       </p>
-      <Hds::Form::Checkbox::Field {{on "change" this.onDocsConfirmChange}} data-test-opt-in-check as |F|>
```

Comment:
FWIW I find this to be a bit easier to read, instead of handling the mutation in the template